cmd/kube-scheduler/app/options/deprecated.go

```diff
@@ -66,7 +65,7 @@ func (o *DeprecatedOptions) AddFlags(fs *pflag.FlagSet, cfg *kubeschedulerconfig
 		"RequiredDuringScheduling affinity is not symmetric, but there is an implicit PreferredDuringScheduling affinity rule corresponding "+
 			"to every RequiredDuringScheduling affinity rule. --hard-pod-affinity-symmetric-weight represents the weight of implicit PreferredDuringScheduling affinity rule. Must be in the range 0-100.")
 	fs.MarkDeprecated("hard-pod-affinity-symmetric-weight", "This option was moved to the policy configuration file")
-	fs.StringVar(&cfg.ResourceProviderClientConnection.Kubeconfig, "resource-providers", cfg.ResourceProviderClientConnection.Kubeconfig, "string representing resource provider kubeconfig files")
+	fs.StringVar(&cfg.ResourceProviderKubeConfig, "resource-providers", cfg.ResourceProviderKubeConfig, "string representing resource provider kubeconfig files")
```

Comment:
so it seem there is no simple way to expand the client connection to be an array of componentbaseconfig.ClientConnectionConfiguration, right ?

Reply:
I tried to make ClientConnectionConfiguration an array. But it did not work for me when assign resource-providers to ClientConnectionConfiguration[0].Kubeconfig